package.json

```diff
@@ -98,14 +98,14 @@
   },
   "scripts": {
     "build": "gatsby build",
-    "build:lambda": "netlify-lambda build src/lambda --config=./webpack.lambda.js",
+    "build:lambda": "NODE_OPTIONS=--openssl-legacy-provider netlify-lambda build src/lambda --config=./webpack.lambda.js",
     "build:10gb": "NODE_OPTIONS=--max-old-space-size=10240 gatsby build",
     "clean": "gatsby clean",
     "crowdin-clean": "rm -rf .crowdin && mkdir .crowdin",
     "crowdin-import": "ts-node src/scripts/crowdin-import.ts",
     "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
     "markdown-checker": "node src/scripts/markdown-checker.js",
-    "generate-heading-ids": "ts-node --esm src/scripts/generateHeadingIds.mts",
```

Comment:
Not sure if it was the new node version or gatsby that didn't like the `.mts` extension so I refactored it to just use `.ts`.